esxi/pyproject.toml

```diff
@@ -37,8 +37,9 @@ dynamic = [
 
 [project.optional-dependencies]
 deps = [
-    "pyvmomi==8.0.2.0.1; python_version > '3.0'",
-    "pysocks==1.7.1"
+    "pysocks==1.7.1",
+    "pyvmomi==8.0.0.1; python_version < '3.0'",
```

Comment:
Where is this change coming from?

Reply:
Not sure about reverting it. My concern is why we didn't have this before. Has something changed in `ddev` recently related to this?

Reply:
not that I can think of tbh.

Reply:
The reason I didn't include the py2 dep before is because ESXi is py2-only

Reply:
Can we leave out the py2 part or does validation fail if we don't have it?

Reply:
I think our script to update deps doesn't take py3-only ints into account. I'll make a card to adjust that. For now I manually removed this.

Reply:
Sounds good, thanks!